.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and publish docs
+
+on:
+  push:
+    branches: ['main']
+    tags: '*'
+
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+concurrency:
+  group: 'pages'
+  cancel-in-progress: false
+
+jobs:
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          filter: tree:0
+          fetch-depth: 0
+
+      - name: Setup Pages
```

Comment:
Add the step to setup Python. Exactly like this one: https://github.com/contentauth/c2pa-python/blob/main/.github/workflows/build.yml#L60-L64